src/csharp/Intel/Iced/Intel/Extended/Label.cs

```diff
@@ -0,0 +1,15 @@
+namespace Iced.Intel
+{
+	public struct Label {
+		public Label(string name, ulong rip) {
+			Name = name;
+			RIP = rip;
+		}
+		
+		public readonly string Name;
+
+		public readonly ulong RIP;
```

Comment:
This isn't a real IP, and it's an implementation detail. It's better to make it `internal` than `public`.

Reply:
That also means the constructor must be internal since it needs `rip`.

Reply:
I have changed the code to use Id and made it internal for now